pkg/api/testing/fuzzer.go

```diff
@@ -357,6 +357,11 @@ func FuzzerFor(t *testing.T, version unversioned.GroupVersion, src rand.Source)
 		},
 		func(pvc *api.PersistentVolumeClaim, c fuzz.Continue) {
 			c.FuzzNoCustom(pvc) // fuzz self without calling this function again
+			pvc.Spec.Selector = &unversioned.LabelSelector{
```

Comment:
Would it make sense to write a custom fuzzer for unversioned.LabelSelector?

I can't tell if this is adding coverage or removing coverage.